src/utils/satelliteUtils.ts

```diff
@@ -19,6 +19,17 @@ export default class SatelliteUtilities {
         return null
     }
 
+    static async getSatellitesByRange(xStart: number, yStart: number, xEnd: number, yEnd: number): Promise<SatelliteDocument[]> {
+        const foundSatellites = await Satellite.find({
+            perigee: { $gt: Math.floor(xStart), $lt: Math.floor(xEnd) },
+            apogee: { $gt: Math.floor(yStart), $lt: Math.floor(yEnd) }
+        }).exec()
+        if (foundSatellites.length === 0) {
```

Comment:
What is the purpose of this if? You are checking if array is empty, and if it is, you are returning a new empty array. I think you could skip this if and just return the array.

Reply:
yes, You're right. Earlier <i was returning some other value I think so that's why this if statement left